OpenOversight/app/templates/incident_new.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'form.html' %}
```

Comment:
On the `/volunteer` page we have these buttons at the bottom - what about we move them to the top, and add a button for "New Incident":

<img width="854" alt="screen shot 2018-05-14 at 9 04 55 pm" src="https://user-images.githubusercontent.com/7832803/40036204-e60e97b8-57ba-11e8-909a-1006a6c16b02.png">